I tried running this but got the following `npm install` error:

```
npm ERR! node v4.2.3
npm ERR! npm  v2.14.7
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react@0.13.1 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer openstax-react-components@0.0.0 wants react@^0.13.1
npm ERR! peerinvalid Peer react-bootstrap@0.23.0 wants react@>=0.13
npm ERR! peerinvalid Peer react-calendar@0.4.0 wants react@^0.13.x
npm ERR! peerinvalid Peer react-datepicker@0.8.0 wants react@>=0.12 <0.14
npm ERR! peerinvalid Peer react-router@0.13.2 wants react@0.13.x
npm ERR! peerinvalid Peer react-scroll-components@0.2.2 wants react@>=0.12.0
npm ERR! peerinvalid Peer fixed-data-table@0.4.1 wants react@>=0.13.0 <0.14.0
npm ERR! peerinvalid Peer react-dom@0.14.7 wants react@^0.14.7
npm ERR! peerinvalid Peer redux-devtools@3.1.1 wants react@^0.14.0
npm ERR! peerinvalid Peer react-redux@4.4.1 wants react@^0.14.0 || ^15.0.0-rc.1
npm ERR! peerinvalid Peer redux-devtools-dock-monitor@1.1.0 wants react@^0.14.0
npm ERR! peerinvalid Peer redux-devtools-log-monitor@1.0.5 wants react@^0.14.0
npm ERR! peerinvalid Peer redux-devtools-inspector@0.3.1 wants react@>=0.13.0
```

Any ideas?

--- a/src/components/app.cjsx
+++ b/src/components/app.cjsx
@@ -3,31 +3,33 @@ BS = require 'react-bootstrap'
 Exercise = require './exercise'
 
 module.exports = React.createClass
-	displayName: 'App'
-	getInitialState: -> {exerciseId: 'new'}
+  displayName: 'App'
+  getInitialState: -> {exerciseId: 'new'}
 
-	update: (event) ->
-		exerciseId = event.target.value
-		@setState({exerciseId})
+  update: (event) ->
+    exerciseId = event.target.value
+    @setState({exerciseId})
 
-	render: ->
-		if @props.id then return <Exercise id={@props.id} />
+  render: ->
+    if @props.id then return <Exercise id={@props.id} />
 
-		<BS.Grid>
-			<BS.Row><BS.Col xs={6}>
-				<p>
-					<label htmlFor="exercise-id">Exercise ID:</label>
-					<input id="exercise-id" type="number" onChange={@update} />
-				</p>
-				<p><a href="/exercises/#{@state.exerciseId}" className="btn btn-primary">
-					Go to Exercise
-				</a></p>
-			</BS.Col><BS.Col xs={6}>
-				<p className="btn btn-success add-exercise">
-					<a href="/exercises/new">
-						<i className="fa fa-plus-circle" />
-						Add New Exercise
-					</a>
-				</p>
-			</BS.Col></BS.Row>
-		</BS.Grid>
+    <BS.Grid>
+      <BS.Row><BS.Col xs={6}>
+        <p>
+          <label htmlFor="exercise-id">Exercise ID:</label>
+          <input id="exercise-id" type="number" onChange={@update} />
+        </p>
+        <p>
+          <a href="/exercises/#{@state.exerciseId}" className="btn btn-primary">
+            Go to Exercise
+          </a>
+        </p>
+      </BS.Col><BS.Col xs={6}>
+        <p className="btn btn-success add-exercise">
+          <a href="/exercises/new">
+            <i className="fa fa-plus-circle" />
+            Add New Exercise
+          </a>
+        </p>
+      </BS.Col></BS.Row>
+    </BS.Grid>

--- a/src/components/app.cjsx
+++ b/src/components/app.cjsx
@@ -1,0 +1,33 @@
+React = require 'react'
+BS = require 'react-bootstrap'
+Exercise = require './exercise'
+
+module.exports = React.createClass
+	displayName: 'App'
+	getInitialState: -> {exerciseId: 'new'}
+
+	update: (event) ->
+		exerciseId = event.target.value
+		@setState({exerciseId})
+
+	render: ->
+		if @props.id then return <Exercise id={@props.id} />
+
+		<BS.Grid>
+			<BS.Row><BS.Col xs={6}>
+				<p>
+					<label htmlFor="exercise-id">Exercise ID:</label>
+					<input id="exercise-id" type="number" onChange={@update} />
+				</p>
+				<p><a href="/exercises/#{@state.exerciseId}" className="btn btn-primary">
+					Go to Exercise
+				</a></p>
+			</BS.Col><BS.Col xs={6}>
+				<p className="btn btn-success add-exercise">
+					<a href="/exercises/new">
+						<i className="fa fa-plus-circle" />
+						Add New Exercise
+					</a>
+				</p>
+			</BS.Col></BS.Row>
+		</BS.Grid>

--- a/src/components/exercise.cjsx
+++ b/src/components/exercise.cjsx
@@ -12,156 +12,156 @@ AttachmentChooser = require './attachment-chooser'
 AsyncButton = require 'openstax-react-components/src/components/buttons/async-button.cjsx'
 
 module.exports = React.createClass
-	displayName: 'Exercise'
+  displayName: 'Exercise'
 
-	getInitialState: -> {}
+  getInitialState: -> {}
 
-	componentWillMount: ->
-		{id} = @props
-		if (id.toLowerCase() is 'new')
-			id = ExerciseStore.freshLocalId()
-			@create(id)
+  componentWillMount: ->
+    {id} = @props
+    if (id.toLowerCase() is 'new')
+      id = ExerciseStore.freshLocalId()
+      @create(id)
 
-		@setState({id})
-		ExerciseStore.addChangeListener(@update)
+    @setState({id})
+    ExerciseStore.addChangeListener(@update)
 
-	update: -> @setState({})
-	updateNumber: (event) -> ExerciseActions.updateNumber(@getId(), event.target?.value)
-	updateStimulus: (event) -> ExerciseActions.updateStimulus(@getId(), event.target?.value)
+  update: -> @setState({})
+  updateNumber: (event) -> ExerciseActions.updateNumber(@getId(), event.target?.value)
+  updateStimulus: (event) -> ExerciseActions.updateStimulus(@getId(), event.target?.value)
 
-	getId: ->
+  getId: ->
     @state.id or @props.id
 
-	getDraftId: (id) ->
-		draftId = if id.indexOf("@") is -1 then id else id.split("@")[0]
-		"#{draftId}@d"
+  getDraftId: (id) ->
+    draftId = if id.indexOf("@") is -1 then id else id.split("@")[0]
+    "#{draftId}@d"
 
-	redirect:(id) ->
-		window.location = "/exercises/#{id}"
+  redirect:(id) ->
+    window.location = "/exercises/#{id}"
 
-	saveExercise: ->
-		validity = ExerciseStore.validate(@getId())
-		if not validity?.valid
-			alert(validity?.reason or 'Not a valid exercise')
-			return
+  saveExercise: ->
+    validity = ExerciseStore.validate(@getId())
+    if not validity?.valid
+      alert(validity?.reason or 'Not a valid exercise')
+      return
 
-		if confirm('Are you sure you want to save?')
-			if ExerciseStore.isNew(@getId())
-				ExerciseStore.on 'created', @redirect
-				ExerciseActions.create(@getId(), ExerciseStore.get(@getId()))
-			else
-				ExerciseActions.save(@getId())
+    if confirm('Are you sure you want to save?')
+      if ExerciseStore.isNew(@getId())
+        ExerciseStore.on 'created', @redirect
+        ExerciseActions.create(@getId(), ExerciseStore.get(@getId()))
+      else
+        ExerciseActions.save(@getId())
 
-	publishExercise: ->
-		if confirm('Are you sure you want to publish?')
-			ExerciseActions.save(@getId())
-			ExerciseActions.publish(@getId())
+  publishExercise: ->
+    if confirm('Are you sure you want to publish?')
+      ExerciseActions.save(@getId())
+      ExerciseActions.publish(@getId())
 
-	renderLoading: ->
-		<div>Loading exercise: {@getId()}</div>
+  renderLoading: ->
+    <div>Loading exercise: {@getId()}</div>
 
-	renderFailed: ->
-		<div>Failed loading exercise, please check id</div>
+  renderFailed: ->
+    <div>Failed loading exercise, please check id</div>
 
-	sync: -> ExerciseActions.sync(@getId())
+  sync: -> ExerciseActions.sync(@getId())
 
-	create: (id) ->
-		template = ExerciseStore.getTemplate()
-		ExerciseActions.loaded(template, id)
+  create: (id) ->
+    template = ExerciseStore.getTemplate()
+    ExerciseActions.loaded(template, id)
 
-	renderForm: ->
-		id = @getId()
+  renderForm: ->
+    id = @getId()
 
-		questions = []
-		for question in ExerciseStore.getQuestions(id)
-			questions.push(<Question key={question.id} sync={@sync} id={question.id} />)
+    questions = []
+    for question in ExerciseStore.getQuestions(id)
+      questions.push(<Question key={question.id} sync={@sync} id={question.id} />)
 
-		isWorking = ExerciseStore.isSaving(id) or ExerciseStore.isPublishing(id)
+    isWorking = ExerciseStore.isSaving(id) or ExerciseStore.isPublishing(id)
 
-		if not ExerciseStore.isPublished(id) and not ExerciseStore.isNew(id)
-			publishButton = <AsyncButton
-				bsStyle='primary'
-				onClick={@publishExercise}
-				disabled={isWorking}
-				isWaiting={ExerciseStore.isPublishing(id)}
-				waitingText='Publishing...'
-				isFailed={ExerciseStore.isFailed(id)}
-				>
-				Publish
-			</AsyncButton>
+    if not ExerciseStore.isPublished(id) and not ExerciseStore.isNew(id)
+      publishButton = <AsyncButton
+        bsStyle='primary'
+        onClick={@publishExercise}
+        disabled={isWorking}
+        isWaiting={ExerciseStore.isPublishing(id)}
+        waitingText='Publishing...'
+        isFailed={ExerciseStore.isFailed(id)}
+        >
+        Publish
+      </AsyncButton>
 
-		<div>
-			<div>
-				<label>Exercise Number</label>: {ExerciseStore.getNumber(id)}
-			</div><div>
-				<label>Exercise Stimulus</label>
-				<textarea onChange={@updateStimulus} defaultValue={ExerciseStore.getStimulus(id)}>
-				</textarea>
-			</div>
-			{questions}
-			<ExerciseTags id={id} />
-			<AsyncButton
-				bsStyle='info'
-				onClick={@saveExercise}
-				disabled={isWorking}
-				isWaiting={ExerciseStore.isSaving(id)}
-				waitingText='Saving...'
-				isFailed={ExerciseStore.isFailed(id)}
-				>
-				Save
-			</AsyncButton>
-			{publishButton}
-		</div>
+    <div>
+      <div>
+        <label>Exercise Number</label>: {ExerciseStore.getNumber(id)}
+      </div><div>
+        <label>Exercise Stimulus</label>
+        <textarea onChange={@updateStimulus} defaultValue={ExerciseStore.getStimulus(id)}>
+        </textarea>
+      </div>
+      {questions}
+      <ExerciseTags id={id} />
+      <AsyncButton
+        bsStyle='info'
+        onClick={@saveExercise}
+        disabled={isWorking}
+        isWaiting={ExerciseStore.isSaving(id)}
+        waitingText='Saving...'
+        isFailed={ExerciseStore.isFailed(id)}
+        >
+        Save
+      </AsyncButton>
+      {publishButton}
+    </div>
 
-	render: ->
-		id = @getId()
+  render: ->
+    id = @getId()
 
-		if not ExerciseStore.get(id) and not ExerciseStore.isFailed(id) and not ExerciseStore.isNew(id)
-			if not ExerciseStore.isLoading(id) then ExerciseActions.load(id)
-			return @renderLoading()
-		else if ExerciseStore.isFailed(id)
-			return @renderFailed()
+    if not ExerciseStore.get(id) and not ExerciseStore.isFailed(id) and not ExerciseStore.isNew(id)
+      if not ExerciseStore.isLoading(id) then ExerciseActions.load(id)
+      return @renderLoading()
+    else if ExerciseStore.isFailed(id)
+      return @renderFailed()
 
-		exercise = ExerciseStore.get(id)
+    exercise = ExerciseStore.get(id)
 
-		exerciseUid = ExerciseStore.getId(id)
-		preview = <Preview exercise={exercise} closePreview={@closePreview}/>
+    exerciseUid = ExerciseStore.getId(id)
+    preview = <Preview exercise={exercise} closePreview={@closePreview}/>
 
-		if ExerciseStore.isPublished(id)
-			publishedLabel =
-				<div>
-					<label>Published: {ExerciseStore.getPublishedDate(id)}</label>
-				</div>
-			editLink =
-				<div>
-					<a href="/exercise/#{@getDraftId(id)}">Edit Exercise</a>
-				</div>
-		else
-			form = @renderForm(id)
+    if ExerciseStore.isPublished(id)
+      publishedLabel =
+        <div>
+          <label>Published: {ExerciseStore.getPublishedDate(id)}</label>
+        </div>
+      editLink =
+        <div>
+          <a href="/exercise/#{@getDraftId(id)}">Edit Exercise</a>
+        </div>
+    else
+      form = @renderForm(id)
 
-		if not ExerciseStore.isNew(id)
-			attachments = <div className="attachments">
-				{ for attachment in ExerciseStore.get(id).attachments
-					<Attachment key={attachment.asset.url} exerciseUid={exerciseUid} attachment={attachment} /> }
-				<AttachmentChooser exerciseUid={exerciseUid} />
-			</div>
-			addExerciseBtn = <p className="btn btn-success add-exercise">
-				<a href="/exercises/new">
-					<i className="fa fa-plus-circle" />Add New Exercise
-				</a>
-			</p>
+    if not ExerciseStore.isNew(id)
+      attachments = <div className="attachments">
+        { for attachment in ExerciseStore.get(id).attachments
+          <Attachment key={attachment.asset.url} exerciseUid={exerciseUid} attachment={attachment} /> }
+        <AttachmentChooser exerciseUid={exerciseUid} />
+      </div>
+      addExerciseBtn = <p className="btn btn-success add-exercise">
+        <a href="/exercises/new">
+          <i className="fa fa-plus-circle" />Add New Exercise
+        </a>
+      </p>
 
-		<BS.Grid>
-			<BS.Row><BS.Col xs={5} className="exercise-editor">
-				<div>
-					<label>Exercise ID:</label> {exerciseUid}
-				</div>
-				{publishedLabel}
-				{editLink}
-				<form>{form}</form>
-			</BS.Col><BS.Col xs={6} className="pull-right">
-				{ addExerciseBtn }
-				{ preview }
-				{ attachments }
-			</BS.Col></BS.Row>
-		</BS.Grid>
+    <BS.Grid>
+      <BS.Row><BS.Col xs={5} className="exercise-editor">
+        <div>
+          <label>Exercise ID:</label> {exerciseUid}
+        </div>
+        {publishedLabel}
+        {editLink}
+        <form>{form}</form>
+      </BS.Col><BS.Col xs={6} className="pull-right">
+        { addExerciseBtn }
+        { preview }
+        { attachments }
+      </BS.Col></BS.Row>
+    </BS.Grid>

--- a/src/components/exercise.cjsx
+++ b/src/components/exercise.cjsx
@@ -12,156 +12,156 @@ AttachmentChooser = require './attachment-chooser'
 AsyncButton = require 'openstax-react-components/src/components/buttons/async-button.cjsx'
 
 module.exports = React.createClass
-  displayName: 'Exercise'
+	displayName: 'Exercise'
 
-  getInitialState: -> {}
+	getInitialState: -> {}
 
-  componentWillMount: ->
-    id = if not @props.id then prompt('Enter exercise id:') else @props.id
-    if (id.toLowerCase() is 'new')
-      id = ExerciseStore.freshLocalId()
-      @create(id)
+	componentWillMount: ->
+		{id} = @props
+		if (id.toLowerCase() is 'new')
+			id = ExerciseStore.freshLocalId()
+			@create(id)
 
-    @setState({id})
-    ExerciseStore.addChangeListener(@update)
+		@setState({id})
+		ExerciseStore.addChangeListener(@update)
 
-  update: -> @setState({})
-  updateNumber: (event) -> ExerciseActions.updateNumber(@getId(), event.target?.value)
-  updateStimulus: (event) -> ExerciseActions.updateStimulus(@getId(), event.target?.value)
+	update: -> @setState({})
+	updateNumber: (event) -> ExerciseActions.updateNumber(@getId(), event.target?.value)
+	updateStimulus: (event) -> ExerciseActions.updateStimulus(@getId(), event.target?.value)
 
-  getId: ->
+	getId: ->
     @state.id or @props.id
 
-  getDraftId: (id) ->
-    draftId = if id.indexOf("@") is -1 then id else id.split("@")[0]
-    "#{draftId}@d"
+	getDraftId: (id) ->
+		draftId = if id.indexOf("@") is -1 then id else id.split("@")[0]
+		"#{draftId}@d"
 
-  redirect:(id) ->
-    window.location = "/exercises/#{id}"
+	redirect:(id) ->
+		window.location = "/exercises/#{id}"
 
-  saveExercise: ->
-    validity = ExerciseStore.validate(@getId())
-    if not validity?.valid
-      alert(validity?.reason or 'Not a valid exercise')
-      return
+	saveExercise: ->
+		validity = ExerciseStore.validate(@getId())
+		if not validity?.valid
+			alert(validity?.reason or 'Not a valid exercise')
+			return
 
-    if confirm('Are you sure you want to save?')
-      if ExerciseStore.isNew(@getId())
-        ExerciseStore.on 'created', @redirect
-        ExerciseActions.create(@getId(), ExerciseStore.get(@getId()))
-      else
-        ExerciseActions.save(@getId())
+		if confirm('Are you sure you want to save?')
+			if ExerciseStore.isNew(@getId())
+				ExerciseStore.on 'created', @redirect
+				ExerciseActions.create(@getId(), ExerciseStore.get(@getId()))
+			else
+				ExerciseActions.save(@getId())
 
-  publishExercise: ->
-    if confirm('Are you sure you want to publish?')
-      ExerciseActions.save(@getId())
-      ExerciseActions.publish(@getId())
+	publishExercise: ->
+		if confirm('Are you sure you want to publish?')
+			ExerciseActions.save(@getId())
+			ExerciseActions.publish(@getId())
 
-  renderLoading: ->
-    <div>Loading exercise: {@getId()}</div>
+	renderLoading: ->
+		<div>Loading exercise: {@getId()}</div>
 
-  renderFailed: ->
-    <div>Failed loading exercise, please check id</div>
+	renderFailed: ->
+		<div>Failed loading exercise, please check id</div>
 
-  sync: -> ExerciseActions.sync(@getId())
+	sync: -> ExerciseActions.sync(@getId())
 
-  create: (id) ->
-    template = ExerciseStore.getTemplate()
-    ExerciseActions.loaded(template, id)
+	create: (id) ->
+		template = ExerciseStore.getTemplate()
+		ExerciseActions.loaded(template, id)
 
-  renderForm: ->
-    id = @getId()
+	renderForm: ->
+		id = @getId()
 
-    questions = []
-    for question in ExerciseStore.getQuestions(id)
-      questions.push(<Question key={question.id} sync={@sync} id={question.id} />)
+		questions = []
+		for question in ExerciseStore.getQuestions(id)
+			questions.push(<Question key={question.id} sync={@sync} id={question.id} />)
 
-    isWorking = ExerciseStore.isSaving(id) or ExerciseStore.isPublishing(id)
+		isWorking = ExerciseStore.isSaving(id) or ExerciseStore.isPublishing(id)
 
-    if not ExerciseStore.isPublished(id) and not ExerciseStore.isNew(id)
-      publishButton = <AsyncButton
-        bsStyle='primary'
-        onClick={@publishExercise}
-        disabled={isWorking}
-        isWaiting={ExerciseStore.isPublishing(id)}
-        waitingText='Publishing...'
-        isFailed={ExerciseStore.isFailed(id)}
-        >
-        Publish
-      </AsyncButton>
+		if not ExerciseStore.isPublished(id) and not ExerciseStore.isNew(id)
+			publishButton = <AsyncButton
+				bsStyle='primary'
+				onClick={@publishExercise}
+				disabled={isWorking}
+				isWaiting={ExerciseStore.isPublishing(id)}
+				waitingText='Publishing...'
+				isFailed={ExerciseStore.isFailed(id)}
+				>
+				Publish
+			</AsyncButton>
 
-    <div>
-      <div>
-        <label>Exercise Number</label>: {ExerciseStore.getNumber(id)}
-      </div><div>
-        <label>Exercise Stimulus</label>
-        <textarea onChange={@updateStimulus} defaultValue={ExerciseStore.getStimulus(id)}>
-        </textarea>
-      </div>
-      {questions}
-      <ExerciseTags id={id} />
-      <AsyncButton
-        bsStyle='info'
-        onClick={@saveExercise}
-        disabled={isWorking}
-        isWaiting={ExerciseStore.isSaving(id)}
-        waitingText='Saving...'
-        isFailed={ExerciseStore.isFailed(id)}
-        >
-        Save
-      </AsyncButton>
-      {publishButton}
-    </div>
+		<div>
+			<div>
+				<label>Exercise Number</label>: {ExerciseStore.getNumber(id)}
+			</div><div>
+				<label>Exercise Stimulus</label>
+				<textarea onChange={@updateStimulus} defaultValue={ExerciseStore.getStimulus(id)}>
+				</textarea>
+			</div>
+			{questions}
+			<ExerciseTags id={id} />
+			<AsyncButton
+				bsStyle='info'
+				onClick={@saveExercise}
+				disabled={isWorking}
+				isWaiting={ExerciseStore.isSaving(id)}
+				waitingText='Saving...'
+				isFailed={ExerciseStore.isFailed(id)}
+				>
+				Save
+			</AsyncButton>
+			{publishButton}
+		</div>
 
-  render: ->
-    id = @getId()
+	render: ->
+		id = @getId()
 
-    if not ExerciseStore.get(id) and not ExerciseStore.isFailed(id) and not ExerciseStore.isNew(id)
-      if not ExerciseStore.isLoading(id) then ExerciseActions.load(id)
-      return @renderLoading()
-    else if ExerciseStore.isFailed(id)
-      return @renderFailed()
+		if not ExerciseStore.get(id) and not ExerciseStore.isFailed(id) and not ExerciseStore.isNew(id)
+			if not ExerciseStore.isLoading(id) then ExerciseActions.load(id)
+			return @renderLoading()
+		else if ExerciseStore.isFailed(id)
+			return @renderFailed()
 
-    exercise = ExerciseStore.get(id)
+		exercise = ExerciseStore.get(id)
 
-    exerciseUid = ExerciseStore.getId(id)
-    preview = <Preview exercise={exercise} closePreview={@closePreview}/>
+		exerciseUid = ExerciseStore.getId(id)
+		preview = <Preview exercise={exercise} closePreview={@closePreview}/>
 
-    if ExerciseStore.isPublished(id)
-      publishedLabel =
-        <div>
-          <label>Published: {ExerciseStore.getPublishedDate(id)}</label>
-        </div>
-      editLink =
-        <div>
-          <a href="/exercise/#{@getDraftId(id)}">Edit Exercise</a>
-        </div>
-    else
-      form = @renderForm(id)
+		if ExerciseStore.isPublished(id)
+			publishedLabel =
+				<div>
+					<label>Published: {ExerciseStore.getPublishedDate(id)}</label>
+				</div>
+			editLink =
+				<div>
+					<a href="/exercise/#{@getDraftId(id)}">Edit Exercise</a>
+				</div>
+		else
+			form = @renderForm(id)
 
-    if not ExerciseStore.isNew(id)
-      attachments = <div className="attachments">
-        { for attachment in ExerciseStore.get(id).attachments
-          <Attachment key={attachment.asset.url} exerciseUid={exerciseUid} attachment={attachment} /> }
-        <AttachmentChooser exerciseUid={exerciseUid} />
-      </div>
-      addExerciseBtn = <p className="btn btn-success add-exercise">
-        <a href="/exercises/new">
-          <i className="fa fa-plus-circle" />Add New Exercise
-        </a>
-      </p>
+		if not ExerciseStore.isNew(id)
+			attachments = <div className="attachments">
+				{ for attachment in ExerciseStore.get(id).attachments
+					<Attachment key={attachment.asset.url} exerciseUid={exerciseUid} attachment={attachment} /> }
+				<AttachmentChooser exerciseUid={exerciseUid} />
+			</div>
+			addExerciseBtn = <p className="btn btn-success add-exercise">
+				<a href="/exercises/new">
+					<i className="fa fa-plus-circle" />Add New Exercise
+				</a>
+			</p>
 
-    <BS.Grid>
-      <BS.Row><BS.Col xs={5} className="exercise-editor">
-        <div>
-          <label>Exercise ID:</label> {exerciseUid}
-        </div>
-        {publishedLabel}
-        {editLink}
-        <form>{form}</form>
-      </BS.Col><BS.Col xs={6} className="pull-right">
-        { addExerciseBtn }
-        { preview }
-        { attachments }
-      </BS.Col></BS.Row>
-    </BS.Grid>
+		<BS.Grid>
+			<BS.Row><BS.Col xs={5} className="exercise-editor">
+				<div>
+					<label>Exercise ID:</label> {exerciseUid}
+				</div>
+				{publishedLabel}
+				{editLink}
+				<form>{form}</form>
+			</BS.Col><BS.Col xs={6} className="pull-right">
+				{ addExerciseBtn }
+				{ preview }
+				{ attachments }
+			</BS.Col></BS.Row>
+		</BS.Grid>

--- a/src/components/exercise.cjsx
+++ b/src/components/exercise.cjsx
@@ -145,9 +145,11 @@ module.exports = React.createClass
           <Attachment key={attachment.asset.url} exerciseUid={exerciseUid} attachment={attachment} /> }
         <AttachmentChooser exerciseUid={exerciseUid} />
       </div>
-      addExerciseBtn = <p className="btn btn-success add-exercise"><a href="/exercises/new">
-        <i className="fa fa-plus-circle" />Add New Exercise
-      </a></p>
+      addExerciseBtn = <p className="btn btn-success add-exercise">
+        <a href="/exercises/new">
+          <i className="fa fa-plus-circle" />Add New Exercise
+        </a>
+      </p>
 
     <BS.Grid>
       <BS.Row><BS.Col xs={5} className="exercise-editor">

--- a/src/index.coffee
+++ b/src/index.coffee
@@ -6,13 +6,13 @@ React = require 'react'
 
 MathJaxHelper =  require 'openstax-react-components/src/helpers/mathjax'
 
-Exercise = require './components/exercise'
+App = require './components/app'
 api = require './api'
 
 
 # Just for debugging
 window.React = React
-window.ExerciseComponent = React.createFactory(Exercise)
+window.AppComponent = React.createFactory(App)
 window.ExerciseActions = ExerciseActions
 window.ExerciseStore = ExerciseStore
 window.AnswerStore = AnswerStore
@@ -37,7 +37,7 @@ loadApp = ->
       id: pathArr[2]
     }
 
-  exercise = window.ExerciseComponent(config)
+  exercise = window.AppComponent(config)
   window.React.render(exercise, root)
 
 document.addEventListener('DOMContentLoaded', loadApp)

